packages/material-ui/src/Switch/Switch.js

```diff
@@ -143,12 +172,15 @@ const Switch = React.forwardRef(function Switch(props, ref) {
         icon={icon}
         checkedIcon={icon}
         classes={{
-          root: clsx(classes.switchBase, classes[`color${capitalize(color)}`]),
+          root: clsx(classes.switchBase, classes[`color${capitalize(color)}`], {
+            [classes[`switchBaseSize${capitalize(size)}`]]: size !== 'medium',
```

Comment:
What do you think of increasing specificity to target this deep element from the root?